### PR TITLE
winging it 5

### PR DESCRIPTION
**Pull request by SondraE**
## Steps to test/reproduce
- Winging It Ep 5 page /2024/03/21/winging-it-05/
- https://www.youtube.com/watch?v=1fSRG8_C2ms

### CODE REVIEW

**Review by jerivas — APPROVED** [review]
LGTM!

**Review by stacyk — COMMENTED** [review]
@SondraE looking good. Left a few consistency comments and a few updates to the front matter that are required.

File: content/blog/wingingit/winging-it-05.md [hunk 1]
```diff
@@ -1,22 +1,23 @@
...
   - FastAPI
+  - Python
+  - SQLAlchemy
 author:
   - ed
-date: 2023-12-21
-length: 30 mins
+  - davidh
+  - james
+date: 2024-03-19
```
Comment (stacyk) [inline content/blog/wingingit/winging-it-05.md:12]: This too will need an update

Comment (SondraE) [reply content/blog/wingingit/winging-it-05.md]: @stacyk So sorry! I had a bunch of those things that you commented on already update, but realized (shortly) after your review) that I hadn't pushed them.

**Review by stacyk — APPROVED** [review]
Looks good. Now we will need to make sure the url on the YouTube show description matches this date (maybe you already have) and it's good to go!

**Pull request merged**